.github/workflows/interop-test.yml

```
@@ -0,0 +1,23 @@
+on: [push, pull_request]
```

If we use this (as we do in https://github.com/libp2p/rust-libp2p/blob/6a9fa3d93008d67ef62eff5b56d5aac3f340fd09/.github/workflows/ci.yml#L3), we wouldn't run the workflow twice on PRs:
```suggestion
on:
  pull_request:
  push:
    branches:
      - master
```